### PR TITLE
add Listen Network info

### PR DESCRIPTION
**Pull request by JimYam**

### CODE REVIEW

**Comment by JimYam** [comment]
We are going to have the next slot auction and there is some information that needs to be added here as soon as possible. We look forward to your merger. thanks!

File: packages/apps-config/src/endpoints/productionRelayKusama.ts [hunk 1]
```diff
@@ -181,6 +181,19 @@ export function createKusama (t: TFunction): EndpointOption {
           OnFinality: 'wss://rpc.api.kico.dico.io'
         }
       },
+
+      {
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/productionRelayKusama.ts:185]:
```suggestion
      {
```

**Comment by JimYam** [comment]
hey, is there any other information we need to refine?

**Comment by jacogr** [comment]
The CI process is failing with a listing issue. So it cannot be merged. 

Running “yarn lint” locally may have an auto fix, depending what the issue is. Otherwise just look at the build logs.

**Comment by JimYam** [comment]
> The CI process is failing with a listing issue. So it cannot be merged.
> 
> Running “yarn lint” locally may have an auto fix, depending what the issue is. Otherwise just look at the build logs.
> 
> Edit: it is an ordering issue, see https://github.com/polkadot-js/apps/runs/5691295343?check_suite_focus=true#step:3:696

This problem appears to be in your master branch, my side to execute `yarn` will report the following error message:
```
/test/apps/.yarn/releases/yarn-3.2.0.cjs:190
`,CHAR_NO_BREAK_SPACE:"\xA0",CHAR_PERCENT:"%",CHAR_PLUS:"+",CHAR_QUESTION_MARK:"?",CHAR_RIGHT_ANGLE_BRACKET:">",CHAR_RIGHT_CURLY_BRACE:"}",CHAR_RIGHT_SQUARE_BRACKET:"]",CHAR_SEMICOLON:";",CHAR_SINGLE_QUOTE:"'",CHAR_SPACE:" ",CHAR_TAB:"	",CHAR_UNDERSCORE:"_",CHAR_VERTICAL_LINE:"|",CHAR_ZERO_WIDTH_NOBREAK_SPACE:"\uFEFF"}});var Lq=w((drt,Dq)=>{"use strict";var OBe=Sy(),{MAX_LENGTH:Rq,CHAR_BACKSLASH:QS,CHAR_BACKTICK:MBe,CHAR_COMMA:UBe,CHAR_DOT:KBe,CHAR_LEFT_PARENTHESES:HBe,CHAR_RIGHT_PARENTHESES:jBe,CHAR_LEFT_CURLY_BRACE:GBe,CHAR_RIGHT_CURLY_BRACE:YBe,CHAR_LEFT_SQUARE_BRACKET:Fq,CHAR_RIGHT_SQUARE_BRACKET:Nq,CHAR_DOUBLE_QUOTE:qBe,CHAR_SINGLE_QUOTE:JBe,CHAR_NO_BREAK_SPACE:WBe,CHAR_ZERO_WIDTH_NOBREAK_SPACE:zBe}=Pq(),_Be=(t,e={})=>{if(typeof t!="string")throw new TypeError("Expected a string");let r=e||{},i=typeof r.maxLength=="number"?Math.min(Rq,r.maxLength):Rq;if(t.length>i)throw new SyntaxError(`Input length (${t.length}), exceeds max characters (${i

SyntaxError: Unexpected token {
    at createScript (vm.js:80:10)
    at Object.runInThisContext (vm.js:139:10)
    at Module._compile (module.js:616:28)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:188:16)
    at bootstrap_node.js:609:3
➜  apps git:(master) 

```

**Comment by jacogr** [comment]
See the CI, log, it runs as expected.

**Pull request closed without merging**